huntsman/utils/__init__.py

```diff
@@ -1,17 +1,41 @@
 import os
 from pocs.utils import listify
 from pocs.utils.config import load_config as config_loader
+from panoptes.utils.config import client
+from huntsman.utils.pyro import get_own_ip
 
+#==============================================================================
 
+def query_config_server(logger=None, *args, **kwargs):
+    '''
+    Query the config server using the IP of the client as the key.
+    '''
+    try:
+        key = get_own_ip()
```

Comment:
Move this out of the try area as if it errors, the error message is specific to `client` failing.